src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -217,6 +245,7 @@ private String createTimerName(final HttpRequest request) {
     private static final ProcessorsV2Factory TELEMETRY_V2_FACTORY = new ProcessorsV2Factory();
     private static final String TELEMETRY_STREAM_V1_PATH = "/telemetry/v1/stream";
     private static final String TELEMETRY_STREAM_V2_PATH = "/telemetry/v2/stream";
+    private static final String SOURCE_PREFIX = "/source/";
```

Comment:
What are we versioning? Collectd api? That seems like a bad idea.  There is an odd coupling between the client and the server versioning here.